utils.h

```diff
@@ -139,4 +147,40 @@ int janus_pidfile_create(const char *file);
 /*! \brief Unlock and remove a previously created PID file
  * @returns 0 if successful, a negative integer otherwise */
 int janus_pidfile_remove(void);
+
+void janus_get_json_type_name(int jtype, gboolean positive_non_empty, char *type_name);
```

Comment:
Can you add a doxygen style documentation for the new methods in this header? You can use the other methods as an example. This would have your new methods automatically be added to the docs, when generated. It doesn't have to be 100% detailed, as I can edit it later on: it's enough to clarify what's what and what the method does in principle.